tests/conftest.py

```diff
@@ -0,0 +1,24 @@
+import pytest
```

Comment:
@mrrajan is this something that must be run before each test?

Reply:
I am not sure which one you are referring to, is it `import pytest`? or the function `setup_before_test`?

Reply:
Just executed against non-aws installation, it works well. Please note, the login screen is designed for keycloak. In order to handle AWS congnito we might need to have configuration and scripts in place.